packages/epo-react-lib/src/atomic/CircularLoader/CircularLoader.tsx

```diff
@@ -0,0 +1,35 @@
+import { FunctionComponent } from "react";
+import IconComposer from "@/svg/IconComposer";
+import { ScreenreaderText } from "@/styles/utils";
+import * as Styled from "./styles";
+
+interface CircularLoaderProps {
+  withOverlay?: boolean;
+  isVisible?: boolean;
+  speed?: Styled.LoaderSpeed;
+  description?: string;
+}
+
+const CircularLoader: FunctionComponent<CircularLoaderProps> = ({
+  withOverlay = false,
+  isVisible = false,
+  speed = "normal",
+  description,
+}) => (
+  <Styled.LoaderContainer
+    {...{ withOverlay, isVisible, speed }}
+    role="status"
+    aria-live="assertive"
+    aria-atomic="true"
+    aria-describedby={description && "loaderDescription"}
```

Comment:
based on the loader accessibility laid out by Carbon: https://carbondesignsystem.com/components/loading/accessibility/